### ISSUE TITLE
[ISSUE] input-file raises Java.lang.ArithmeticException: / by zero if file is not found

### ISSUE THREAD

**Issue by sonots** [embulk/embulk#382]
```
in:
  type: file
  path_prefix: example/example.foo
```

WIth 0.7.9

```
2016-01-27 21:58:24.022 +0900: Embulk v0.7.9
2016-01-27 21:58:25.420 +0900 [INFO] (transaction): Loaded plugin embulk/filter/row from a load path
2016-01-27 21:58:25.445 +0900 [INFO] (transaction): Listing local files at directory 'example' filtering filename by prefix 'example.foo'
2016-01-27 21:58:25.450 +0900 [INFO] (transaction): Loading files []
2016-01-27 21:58:25.527 +0900 [INFO] (main): Committed.
2016-01-27 21:58:25.527 +0900 [INFO] (main): Next config diff: {"in":{},"out":{}}
```

With 0.8.1

```
org.embulk.exec.PartialExecutionException: java.lang.ArithmeticException: / by zero                                                                                                             [180/1372]
        at org.embulk.exec.BulkLoader$LoaderState.buildPartialExecuteException(org/embulk/exec/BulkLoader.java:363)
        at org.embulk.exec.BulkLoader.doRun(org/embulk/exec/BulkLoader.java:572)
        at org.embulk.exec.BulkLoader.access$000(org/embulk/exec/BulkLoader.java:33)
        at org.embulk.exec.BulkLoader$1.run(org/embulk/exec/BulkLoader.java:374)
        at org.embulk.exec.BulkLoader$1.run(org/embulk/exec/BulkLoader.java:370)
        at org.embulk.spi.Exec.doWith(org/embulk/spi/Exec.java:25)
        at org.embulk.exec.BulkLoader.run(org/embulk/exec/BulkLoader.java:370)
        at org.embulk.EmbulkEmbed.run(org/embulk/EmbulkEmbed.java:180)
        at java.lang.reflect.Method.invoke(java/lang/reflect/Method.java:606)
        at RUBY.run(/usr/local/Cellar/embulk/0.7.10/bin/embulk!/embulk/runner.rb:77)
        at RUBY.run(/usr/local/Cellar/embulk/0.7.10/bin/embulk!/embulk/command/embulk_run.rb:296)
        at RUBY.<top>(/usr/local/Cellar/embulk/0.7.10/bin/embulk!/embulk/command/embulk_main.rb:2)
        at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:939)
        at RUBY.(root)(uri:classloader:/META-INF/jruby.home/lib/ruby/stdlib/rubygems/core_ext/kernel_require.rb:1)
        at usr.local.Cellar.embulk.$0_dot_7_dot_10.bin.embulk.embulk.command.embulk_bundle.<top>(file:/usr/local/Cellar/embulk/0.7.10/bin/embulk!/embulk/command/embulk_bundle.rb:51)
        at java.lang.invoke.MethodHandle.invokeWithArguments(java/lang/invoke/MethodHandle.java:599)
        at org.embulk.cli.Main.main(org/embulk/cli/Main.java:23)
        Suppressed: java.lang.NullPointerException
                at org.embulk.exec.BulkLoader.doCleanup(BulkLoader.java:479)
                at org.embulk.exec.BulkLoader$3.run(BulkLoader.java:410)
                at org.embulk.exec.BulkLoader$3.run(BulkLoader.java:406)
                at org.embulk.spi.Exec.doWith(Exec.java:25)
                at org.embulk.exec.BulkLoader.cleanup(BulkLoader.java:406)
                at org.embulk.EmbulkEmbed.run(EmbulkEmbed.java:184)
                at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
                at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
                at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
                at java.lang.reflect.Method.invoke(Method.java:606)
                at org.jruby.javasupport.JavaMethod.invokeDirectWithExceptionHandling(JavaMethod.java:452)
                at org.jruby.javasupport.JavaMethod.invokeDirect(JavaMethod.java:313)
                at org.jruby.java.invokers.InstanceMethodInvoker.call(InstanceMethodInvoker.java:45)
                at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:313)
                at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:163)
                at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:290)
                at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:77)
                at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:127)
                at org.jruby.internal.runtime.methods.MixedModeIRMethod.call(MixedModeIRMethod.java:113)
                at org.jruby.runtime.callsite.CachingCallSite.cacheAndCall(CachingCallSite.java:273)
                at org.jruby.runtime.callsite.CachingCallSite.callBlock(CachingCallSite.java:79)
                at org.jruby.runtime.callsite.CachingCallSite.call(CachingCallSite.java:83)
                at org.jruby.ir.instructions.CallBase.interpret(CallBase.java:419)
                at org.jruby.ir.interpreter.InterpreterEngine.processCall(InterpreterEngine.java:322)
                at org.jruby.ir.interpreter.StartupInterpreterEngine.interpret(StartupInterpreterEngine.java:77)
                at org.jruby.ir.interpreter.InterpreterEngine.interpret(InterpreterEngine.java:83)
                at org.jruby.internal.runtime.methods.MixedModeIRMethod.INTERPRET_METHOD(MixedModeIRMethod.java:197)
```

**Comment by hiroyuki-sato** [comment]
Maybe this is same issue  https://github.com/embulk/embulk/pull/375 
And already fixed on master. (not yet released).

**Comment by sonots** [comment]
okay, thank. I will try the patch.

**Comment by sonots** [comment]
It seems this issue has been fixed with #375. I will close, thanks.
